ci/docker/image-builder/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+
```

Comment:
Here is a CLI app that builds Docker images. 

As before, it's completely independent of any code in the repository.
Otherwise, we should rebuild all images on any change in the repository to be sure.